tests/keepalived_haproxy_master_example.yml

```diff
@@ -128,10 +128,74 @@ keepalived_instances:
 #              warmup: true
 #              # Optinal, set false or omit to not use it.
 #              misc_dynamic: true
+#              # Optional, Specify the username/groupname that the script should
+#                          be run under.
+#                          If group is not specified, the group of the user
+#                          is used
+#              user: 'root'
+#              group: 'root'
 #        ssl_get:
 #          - url_path: '/'
 #            url_digest: 'a797b47875d8fd5c949066182902099d'
 #            # Optional
 #            connect_timeout: 3
 #            nb_get_retry: 3
 #            delay_before_retry: 2
+#            # Ip to connect to, if not set real_server ip will be used
```

Comment:
Could you add `(Optional)` or `Optional,` at the beggining of the comment, please? That would be awesome.

Reply:
sure. i'll check my comments again and update it accordingly

Reply:
Perfect!